src/sdk/pynni/nni/compression/torch/pruners.py

```diff
@@ -198,8 +197,9 @@ def __init__(self, model, config_list):
         all_bn_weights = torch.cat(weight_list)
         k = int(all_bn_weights.shape[0] * config['sparsity'])
         self.global_threshold = torch.topk(all_bn_weights.view(-1), k, largest=False)[0].max()
+        self.register_buffer("if_calculated", torch.tensor(False)) # pylint: disable=not-callable
```

Comment:
Is this `if_calculated` buffer shared by all layers?   In previous version, there are flags for each layer in the dict.

Reply:
`if_calculated` buffer is registered per wrapper module.

Reply:
Ok